fitbenchmarking/controllers/base_controller.py

```diff
@@ -21,6 +22,9 @@ class Controller:
 
     VALID_FLAGS = [0, 1, 2, 3, 4, 5, 6]
 
+    # Invalid problem formats for specific jacobian methods
+    INVALID_JACOBIAN_FORMATS = {"cs": ["mantid"]}
```

Comment:
With this being in the base class I'm concerned about name clashes. E.g. if another jacobian software is added with a method named "cs".

Reply:
I like the new implementation with the attributes on the relevent controller but the problem of having the same method on multiple jacobian classes could still be an issue